pennylane/ops/op_math/__init__.py

```diff
@@ -22,7 +22,7 @@
 
 from .adjoint_class import Adjoint
 from .adjoint_constructor import adjoint
-from .sum import sum, Sum
+from .sum import op_sum, Sum  # top level func cannot be cva
```

Comment:
Cva?

Reply:
Thats weird, I don't remember adding that comment. I did change the name tho since it mimics the built in `sum` method, and we probably don't want to overwrite that.

Reply:
Maybe remove the comment then? It will probably cause confusion in the future.